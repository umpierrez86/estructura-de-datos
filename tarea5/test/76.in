# Prueba de rotar
rotar 20 LR
insertarEnAbb (50,0)
rotar 20 LR
rotar 50 LR
rotar 50 LL
insertarEnAbb (30,0)
insertarEnAbb (40,0)
insertarEnAbb (45,0)
rotar 50 LL
imprimirAbb
rotar 30 RR
imprimirAbb
rotar 50 LR
imprimirAbb
rotar 40 RL
imprimirAbb
rotar 30 LL
imprimirAbb
rotar 40 LR
imprimirAbb
insertarEnAbb (35,0)
rotar 40 LR
imprimirAbb
insertarEnAbb (70,0)
insertarEnAbb (60,0)
insertarEnAbb (65,0)
insertarEnAbb (90,0)
insertarEnAbb (80,0)
insertarEnAbb (55,0)
imprimirAbb
rotar 70 RL
imprimirAbb
rotar 80 LL
imprimirAbb
rotar 70 RR
imprimirAbb
rotar 50 RL
imprimirAbb
rotar 70 LR
imprimirAbb
Fin