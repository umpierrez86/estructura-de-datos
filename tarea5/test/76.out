1>#  Prueba de rotar.
2>
3>Se insertó (50,0.00) en el abb.
4>
5>
6>
7>Se insertó (30,0.00) en el abb.
8>Se insertó (40,0.00) en el abb.
9>Se insertó (45,0.00) en el abb.
10>
11>
-(50,0.00)
---(45,0.00)
--(40,0.00)
(30,0.00)
12>
13>
(50,0.00)
---(45,0.00)
--(40,0.00)
-(30,0.00)
14>
15>
-(50,0.00)
--(45,0.00)
(40,0.00)
-(30,0.00)
16>
17>
-(50,0.00)
(45,0.00)
-(40,0.00)
--(30,0.00)
18>
19>
-(50,0.00)
(45,0.00)
-(40,0.00)
--(30,0.00)
20>
21>
-(50,0.00)
(45,0.00)
-(40,0.00)
--(30,0.00)
22>Se insertó (35,0.00) en el abb.
23>
24>
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
25>Se insertó (70,0.00) en el abb.
26>Se insertó (60,0.00) en el abb.
27>Se insertó (65,0.00) en el abb.
28>Se insertó (90,0.00) en el abb.
29>Se insertó (80,0.00) en el abb.
30>Se insertó (55,0.00) en el abb.
31>
---(90,0.00)
----(80,0.00)
--(70,0.00)
----(65,0.00)
---(60,0.00)
----(55,0.00)
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
32>
33>
---(90,0.00)
--(80,0.00)
---(70,0.00)
-----(65,0.00)
----(60,0.00)
-----(55,0.00)
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
34>
35>
----(90,0.00)
---(80,0.00)
--(70,0.00)
----(65,0.00)
---(60,0.00)
----(55,0.00)
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
36>
37>
---(90,0.00)
--(80,0.00)
---(70,0.00)
-----(65,0.00)
----(60,0.00)
-----(55,0.00)
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
38>
39>
---(90,0.00)
--(80,0.00)
-(70,0.00)
----(65,0.00)
---(60,0.00)
----(55,0.00)
--(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
40>
41>
----(90,0.00)
---(80,0.00)
--(70,0.00)
---(65,0.00)
-(60,0.00)
---(55,0.00)
--(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
42>Fin.
